app/src/components/main/grid/propTypes.js

```diff
@@ -3,8 +3,8 @@ import { ALIGN_CENTER, ALIGN_LEFT, ALIGN_RIGHT } from './constants';
 
 export const columnPropTypes = {
   title: PropTypes.shape({
-    full: PropTypes.string,
-    short: PropTypes.string,
+    full: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
```

Comment:
Looks like this change is not necessary for now. See comments below.

Reply:
Reverted